src/controllers/ioWalletController.ts

```diff
@@ -125,6 +126,48 @@ export default class IoWalletController {
       )()
     );
 
+  /**
+   * Update current Wallet Instance status.
+   */
+  public readonly setWalletInstanceStatus = (
+    req: express.Request
+  ): Promise<
+    | IResponseErrorInternal
+    | IResponseSuccessNoContent
+    | IResponseErrorServiceUnavailable
+    | IResponseErrorValidation
+    | IResponseErrorForbiddenNotAuthorized
+  > =>
+    withUserFromRequest(req, async (user) =>
+      pipe(
+        this.ensureFiscalCodeIsAllowed(user.fiscal_code),
```

Comment:
I think this is not necessary since a non-enabled user will certainly not have Wallet Instances on the DB